Doc/c-api/unicode.rst

```diff
@@ -1597,6 +1612,20 @@ They all return ``NULL`` or ``-1`` if an exception occurs.
       Strings interned this way are made :term:`immortal`.
 
 
+.. c:function:: unsigned int PyUnicode_CHECK_INTERNED(PyObject *str)
+
+   Return a non-zero value if *str* is interned, zero if not.
```

Comment:
Most documentation uses "Return true" (60 occurrences), some use "Return non-zero" (13 occurrences) and one uses "Return a non-zero".

In this case using "Return a non-zero" looks justified, as it may encode additional information.